### PR TITLE
Adds docker ci

### PR DESCRIPTION
**Pull request by mholthausen**
This PR

- adds basic eslint
- adds a deploy GitHubAction to build and deploy the workers as Docker images to `ghcr.io/klips-project/mqm-worker/`
  - uses `github.actor` (the user that initiated the workflow run) and the `GITHUB_TOKEN` as credentials
- modifies the Dockerfiles for each worker for automatic deploy by GitHubActions
- adds a test GitHubAction to install and test (eslint) the source code
- adjusts the README file
- cleans up the repository with unneeded files

@weskamm please review

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+env:
+  browser: true
+  commonjs: true
```
Comment (weskamm) [inline .eslintrc.yml:3]: why do we need this?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        uses: actions/checkout@v2.3.4
+        with:
+          persist-credentials: false
+
+      - name: Install and Build 🔧
+        uses: actions/setup-node@v2.1.5
+      - run: npm ci
+      - run: npm run test
+
+      - name: Set up QEMU 🔧
```
Comment (weskamm) [inline .github/workflows/deploy.yml:21]: why dont we publish to github, as tested here: https://github.com/klips-project/ghcr-ci-getstarted/tree/main/.github/workflows

Comment (mholthausen) [reply .github/workflows/deploy.yml]: Removed QEMU. It is used to build against other platforms, which is not needed here yet.

File: .eslintrc.yml (hunk 1 above)
Comment (mholthausen) [reply .eslintrc.yml]: These configurations are not needed at all and have been removed.

**Comment by mholthausen** [comment]
Thanks for the review. All issues have been resolved. Will merge soon.

**Pull request merged**